lib/rules/selector-max-specificity/__tests__/index.mjs

```diff
@@ -439,65 +439,77 @@ testRule({
 	config: [
 		'0,1,0',
 		{
-			ignoreSelectors: [':global', ':local', '/my-/'],
+			ignoreSelectors: [':is', ':has', ':host', '/my-/'],
```

Comment:
question

Do we have something covering https://chromium.googlesource.com/chromium/src/+/674ef6ac2499fb51d82e752ccf723fd76c505caf/third_party/WebKit/LayoutTests/external/wpt/css/css-scoping/host-context-specificity-003.html?
ref https://chromestatus.com/feature/5177305701810176

Reply:
Yes, but upstream : https://github.com/csstools/postcss-plugins/blob/3ac6363e487f6e22637c46107498d1ece1fa8692/packages/selector-specificity/test/tests-from-bramus-specificity.mjs#L111-L116